scripts/import/import_variant_synonyms

```diff
@@ -165,6 +167,45 @@ elsif($source_name eq 'UniProt'){
   add_synonyms_uniprot(\%synonym, $source_name, $variation_set_id, $dba, $species); 
 }
 
+elsif($source_name eq 'EVA') {
+  my $result_synonyms = parse_eva_file($data_file);
+
+  # Source EVA
+  my %source_eva = (
+  "EVA" => {
+    description => "Short variant data imported from EVA",
+    url         => "https://www.ebi.ac.uk/eva/",
+    status      => "germline",
+  }, 
+  );
+
+  # get source info 
+  $source_description ||= $source_eva{$source_name}->{description};
+  $source_url ||= $source_eva{$source_name}->{url};
+  my $source_status = $source_eva{$source_name}->{status};
+
+  my $source = get_source($species, $dba, $source_name, $source_version, $source_url, $source_description, $source_status);
+
+  import_synonyms($result_synonyms, $source, $dba, $species);
+}
+
+elsif($source_name eq 'pig_chip') {
```

Comment:
Why having a specific parameter for pig? Can't this be generalised for any species — if needed for the future?

Reply:
This method is getting data only from pig_synonym, it's specific to pig only.